tests/unit/test_sqlalchemy_utils.py

```diff
@@ -1,9 +1,13 @@
+from unittest import mock
+
 import pytest
 import sqlalchemy
 from sqlalchemy.dialects.sqlite.pysqlite import SQLiteDialect_pysqlite
 
 from databuilder.sqlalchemy_utils import (
+    ExecuteWithRetry,
     GeneratedTable,
+    OperationalError,
```

Comment:
I think because I'm treating it semantically as "the kind of error that ExecuteWithRetry looks for, whatever that kind may be" rather than caring what specific sort of error it is. I guess you could argue that either way though.